src/third-parties/google-maps-embed/data.json

```diff
@@ -8,6 +8,7 @@
       "loading": "lazy",
       "src": {
         "url": "https://www.google.com/maps/embed/v1/place",
```

Comment:
Should this be  "https://www.google.com/maps/embed/v1/"

So when we pass slugParam it would be appended correctly

Reply:
I set up `slugParam` to replace the last slug so that `/place` is the default, but I can change the logic if that would make more sense? This way, without specifying `mode` will still result in this default URL

Reply:
Yes, that sounds better